hermes-api/src/main/java/pl/allegro/tech/hermes/api/MaintainerDescriptor.java

```diff
@@ -0,0 +1,53 @@
+package pl.allegro.tech.hermes.api;
+
+import com.fasterxml.jackson.annotation.JsonCreator;
+import com.fasterxml.jackson.annotation.JsonProperty;
+
+import javax.validation.constraints.NotNull;
+import java.util.Objects;
+
+public final class MaintainerDescriptor {
```

Comment:
Well, if you both don't feel comfortable with `Descriptor` then it obviously doesn't serve it's purpose. `MaintainerId` is fine enough.